core/field_colour_slider.js

```diff
@@ -45,343 +35,307 @@ goog.require('goog.ui.Slider');
  * @extends {Blockly.Field}
  * @constructor
  */
-Blockly.FieldColourSlider = function(colour, opt_validator) {
-  Blockly.FieldColourSlider.superClass_.constructor.call(this, colour, opt_validator);
-  this.addArgType('colour');
-
-  // Flag to track whether or not the slider callbacks should execute
-  this.sliderCallbacksEnabled_ = false;
-};
-goog.inherits(Blockly.FieldColourSlider, Blockly.Field);
-
-/**
- * Construct a FieldColourSlider from a JSON arg object.
- * @param {!Object} options A JSON object with options (colour).
- * @returns {!Blockly.FieldColourSlider} The new field instance.
- * @package
- * @nocollapse
- */
-Blockly.FieldColourSlider.fromJson = function(options) {
-  return new Blockly.FieldColourSlider(options['colour']);
-};
-
-/**
- * Function to be called if eyedropper can be activated.
- * If defined, an eyedropper button will be added to the color picker.
- * The button calls this function with a callback to update the field value.
- * BEWARE: This is not a stable API, so it is being marked as private. It may change.
- * @private
- */
-Blockly.FieldColourSlider.activateEyedropper_ = null;
-
-/**
- * Path to the eyedropper svg icon.
- */
-Blockly.FieldColourSlider.EYEDROPPER_PATH = 'eyedropper.svg';
-
-/**
- * Install this field on a block.
- * @param {!Blockly.Block} block The block containing this field.
- */
-Blockly.FieldColourSlider.prototype.init = function(block) {
-  if (this.fieldGroup_) {
-    // Colour slider has already been initialized once.
-    return;
+export class FieldColourSlider extends Blockly.FieldColour {
```

Comment:
Missed this yesterday, we're removing the colour field from core b/c reasons, so might as well just start using the plugin one.

https://www.npmjs.com/package/@blockly/field-colour

Reply:
Done!